src/ig/IGExporter.ts

```diff
@@ -195,21 +196,64 @@ export class IGExporter {
   }
 
   /**
-   * SUSHI doesn't yet support authors adding additional pages beyond index.md.  If we detect
-   * additional pages, we need to log an error.
+   * Adds additional pages beyond index.md that are defined by the user.
+   * Only add formats that are supported by the IG template
+   * Intro and notes file contents are injected into relevant pages and should not be treated as their own page
+   *
+   * @param igPath {string} - the path where the IG is exported to
    */
-  private checkForOtherPageContent() {
+  private addOtherPageContent(igPath: string) {
     const inputPageContentPath = path.join(this.igDataPath, 'input', 'pagecontent');
     if (fs.existsSync(inputPageContentPath)) {
-      const pages = fs.readdirSync(inputPageContentPath);
-      if (pages.length > 1 || (pages.length === 1 && pages[0] !== 'index.md')) {
-        logger.error('SUSHI does not yet support custom pagecontent other than index.md.', {
+      const pages = fs
+        .readdirSync(inputPageContentPath)
+        .filter(page => page !== 'index.md')
```

Comment:
I think we probably need to add special handling for `index.xml` too.  So this should really be:
```
        .filter(page => page !== 'index.md' && page !== 'index.xml')
```

In addition, we should update the `addIndex` function to check for an `index.xml` too and not generate an index if it already exists.  And to make sure we put in the right `generation` value depending on if the index is `index.md` or `index.xml`.

Reply:
Sorry for putting all that here, but since `addIndex` isn't changed in this PR, GitHub doesn't make it easy for me to comment on it there!